README.md

```diff
@@ -92,9 +97,89 @@ Given the approach and requirements above, the Principal Surprise (PS) algorithm
 
       1. Also compute the `abs` individual contribution of each task-type surprise to the distance from mean for each worker to rank task-types by abnormality per-worker
 
-#### Example Use Case
-
-See `examples/transport.csv`
+### Real-World Data
+
+As part of an Ona partner project, a set of field worker task data was analyzed with many (dozens) of tasks types.  A sample is here:
+
+![Task 1 Distribution](/home/gstuder/Workspaces/Ona/community-health-task-investigator/docs/images/task_time_dist_1.png)
```

Comment:
I don't have a good process for getting these links working - do I need to edit them in the PR, or do I need to wait until stuff is merged to master so the image link is stable?

Reply:
what I do is push something that is broken until after merge, for this

https://raw.githubusercontent.com/onaio/community-health-task-investigator/master/docs/images/task_time_dist_1.png